backend/s3/s3.go

```diff
@@ -3173,6 +3206,8 @@ func setQuirks(opt *Options) {
 	case "LyveCloud":
 		useMultipartEtag = false // LyveCloud seems to calculate multipart Etags differently from AWS
 		useAlreadyExists = false // untested
+	case "Magalu":
```

Comment:
Did you run the integration tests to check these are the correct quirks?

Reply:
Great - well done.

The Multithread Copy errors are most likely a bug in the service.

Reply:
In my view, the pr is ready, does it need anything else @ncw ?